portfolio/src/main/webapp/comments.js

```diff
@@ -80,6 +93,19 @@ class Comments {
       throw Error(`${response.statusText}. Status: ${response.status}`);
     }
   }
+
+  /**
+   * Returns an error if the response status is not between 200 and 299, i.e not OK.
+   * @param {*} response - The HTTP response received from the servlet.
+   * @return {*} - If an error is not thrown, returns the response (not JSON).
+   */
+  checkPostError(response) {
+    if (response.ok) {
+      return;
+    } else {
+      throw Error(`${response.statusText}. Status: ${response.status}`);
```

Comment:
Add an error message about where this failed.

```
Error occurred while deleting comments.
```

Reply:
I did this in a branch higher up the stack.

Reply:
I added an error message in each catch() statement describing where it occurred since it's impossible to know why an error occured from within the checkFetchError and checkPostError method alone.